squid_control/control/config.py

```diff
@@ -581,17 +591,14 @@ def read_config(self, config_path):
 
 def load_config(config_path, multipoint_function):
     global CONFIG
-    home_dir = Path.home()
-    config_dir = home_dir / '.squid-control'
 
-    # Ensure the .squid-control directory exists
-    config_dir.mkdir(exist_ok=True)
+    config_dir = os.path.abspath(__file__)
+    config_dir = Path(config_dir)
```

Comment:
I would merge the two lines into one, and why removing `config_dir.mkdir(exist_ok=True)`?